src/script/index.js

```
@@ -1 +1,132 @@
 import '../styles/main.scss';
+let hamIcon = document.getElementById('hamIcon');
+let crossIcon = document.getElementById('crossIcon');
+const navBtns = document.querySelector('.nav-btns');
+const hamMenu = document.getElementById('hamMenu');
+const navParent = document.querySelector('.nav-parent');
+const navMenu = document.querySelector('.nav-links');
+const navLinks = document.querySelectorAll('.nav-links li a');
+const navButtons = document.querySelectorAll('.nav-btns button');
+hamIcon.addEventListener('click', () => {
+    hamMenu.style.width = window.innerWidth / 2 + 'px';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)